commands.sh

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/sh
+# You will need to set this variable to match your local directory structure
+# TDRS_HOME="$HOME/Where/Ever/You/Want/TANF-app"
+
+# navigate terminal to tdrs home if $TDRS_HOME is set
+alias cd-tdrs='cd "$TDRS_HOME"'
+
+# navigate terminal to tdrs frontend if $TDRS_HOME is set
+alias cd-tdrs-frontend='cd "$TDRS_HOME/tdrs-frontend"'
+
+# navigate terminal to tdrs backend if $TDRS_HOME is set
+alias cd-tdrs-backend='cd "$TDRS_HOME/tdrs-backend"'
+
+# shortcut for applying all relavent compose files for local development
+alias tdrs-compose-local='docker-compose -f docker-compose.yml -f docker-compose.local.yml'
+
+# Stop tdrs backend entirely, then start it up again
+alias tdrs-backend-hard-restart='tdrs-start-backend && tdrs-start-backend'
+
+# shortcut for running bash commands in backend container
+alias tdrs-backend-exec='tdrs-backend-compose exec web /bin/bash'
+
+# Open shell_plus for django backend inside of container
+alias tdrs-django-shell='tdrs-backend-compose run --rm web bash -c "python manage.py shell_plus"'
+
+# TDRS Frontend aliases
+
+# start both the frontend and backend
+alias start-tdrs='tdrs-start-frontend && tdrs-start-backend'
+
+# Stop both the frontend and the backend
+alias stop-tdrs='tdrs-stop-frontend && tdrs-stop-backend'
+
+# Restart frontend and backend
+alias tdrs-restart='tdrs-restart-frontend && tdrs-restart-backend'
+
```

Comment:
Could use some more explaination on this one @abottoms-coder

Reply:
I recall I discussing this in a call but for posteriority:

This alias just lists out all aliases *and* shell functions with the "tdrs" prefix so people can easily find all and use them. I was using something similar for my testing and figured it might be helpful if someone is forgetting the exact wording.

Reply:
Resolved at end of file.